csaf-import/src/main/kotlin/io/github/csaf/sbom/retrieval/RetrievedAggregator.kt

```diff
@@ -25,4 +26,28 @@ import io.github.csaf.sbom.schema.generated.Aggregator
  *
  * This class is not yet implemented.
```

Comment:
doc needs to be adjusted

Reply:
It's true, current implementation is bare minimum to have something to test. But will change wording.